pom.xml

```diff
@@ -306,6 +316,11 @@
           <artifactId>maven-resources-plugin</artifactId>
           <version>3.2.0</version>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-dependency-plugin</artifactId>
+          <version>3.1.2</version>
+        </plugin>
```

Comment:
The maven-dependency-plugin 2.8 was showing google-api-client-java6 does not use google-api-client. However, the module does not compile without google-api-client:

[ERROR] /Users/suztomo/google-api-java-client/google-api-client-java6/src/main/java/com/google/api/client/googleapis/extensions/java6/auth/oauth2/GooglePromptReceiver.java:[34,12] cannot find symbol
[ERROR]   symbol:   variable GoogleOAuthConstants
[ERROR]   location: class com.google.api.client.googleapis.extensions.java6.auth.oauth2.GooglePromptReceive

Therefore, the version of the dependency plugin showing false positive.

The newer version of maven-dependency-plugin shows the correct result.